modules/contract/contracts.js

```diff
@@ -5,66 +5,94 @@ var config = jsonfile.readFileSync(confFile);
-        console.log(Helper.outputInfo(`If you're using ${TESTNET_NAME} node, use https://${TESTNET_NAME}.tzstats.com to check contract/transactions`));
+        var parseError = result.indexOf('Instead, ');
+        Logger.info(result.substring(0, parseError != -1  ? parseError : result.length));
+        Logger.info(Helper.outputInfo(`If you're using ${TESTNET_NAME} node, use https://${TESTNET_NAME}.tzstats.com to check contract/transactions`));
     }
 
     async getStorage(args) {
+        Logger.verbose(`Command : tezster get-storage ${args}`);
         if (args.length < 1) {
-            console.log(Helper.outputInfo('Incorrect usage of get-storage command \nCorrect usage: - tezster get-storage <contract-name>'));
+            Logger.info(Helper.outputInfo('Incorrect usage of get-storage command \nCorrect usage: - tezster get-storage <contract-name>'));
             return;
         }
-        await this.loadTezsterConfig(); 
         
         let result = await this.getContractStorage(args[0]);
-        console.log(result);
+        Logger.info(result);
     }
 
     async addContract(args) {
+        Logger.verbose(`Command : tezster add-contract ${args}`);
         if (args.length < 2) {
-            return console.log(Helper.outputError('Incorrect usage - tezster add-contract <label> <Address>'));
+            return Logger.info(Helper.outputError('Incorrect usage - tezster add-contract <label> <Address>'));
+        }
+        Logger.info(this.addContractToConfig(args[0], args[1], ''));      
+    }
+
+    async getEntryPoints(args) {
+        Logger.verbose(`Command : tezster extract-entry-points ${args}`);
+        if (args.length < 1) {
+            return Logger.info(Helper.outputError('Incorrect usage - tezster list-entry-points <contract-absolute-path>'));
+        }
+        this.extractEntryPoints(args[0]);
+    }
+
+    async extractEntryPoints(contractPath) {
+        const fs = require('fs');
+        const conseiljs = require(ConseilJS);
+        try {
+            const contractCode = fs.readFileSync(contractPath, 'utf8');
+            const entryPoints = await conseiljs.TezosContractIntrospector.generateEntryPointsFromCode(contractCode);
+            const storageFormat = await conseiljs.TezosLanguageUtil.preProcessMichelsonScript(contractCode);
+            Logger.info(`Initial Storage input must be of type : ${storageFormat[1].slice(8)}`);
+            entryPoints.forEach(p => {
+                Logger.info(`\nName => ${p.name}\nParameters => (${p.parameters.map(pp => (pp.name || '') + pp.type).join(', ')})\nStructure => ${p.structure}\n`);
+            });
+        }
+        catch(error) {
+            return Helper.outputError(error);
         }
-        await this.loadTezsterConfig();
-        console.log(this.addContractToConfig(args[0], args[1], ''));      
     }
 
-    async deploy(contractLabel, contractPath, initValue, account) {
+    async deploy(contractLabel, contractPath, initValue, account, amount) {
```

Comment:
you can use amount = 0 to make amount defaulted to 0. you don't require L117 block then.